scripts/download-rc.js

```diff
@@ -56,8 +56,9 @@ const downloadRC = () => {
                 shell: ON_WINDOWS
             });
         if (subprocess.status !== 0) {
+            let subprocessTrace = subprocess.error ? subprocess.error.stack : '';
```

Comment:
I added this after trying to execute integration test on a new machine. Mvn was not installed and the error printed to console was not helpful. This prints the stack if there is an error related to starting mvn process

Reply:
That's a good improvement.

BTW looks like this variable could be a `const`

Reply:
right it was different before, I forgot to change. thanks